staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -158,11 +162,46 @@ func getExcludedChecks(r *http.Request) sets.String {
 	return sets.NewString()
 }
 
+type verbosityLevel int
+
+const (
+	quiet verbosityLevel = iota
+	verbose
+	duration
+)
+
+// verbosity returns the level of verbosity for health check.
```

Comment:
An alternative approach might to provide a structured data healthz output (e.g. json) which can safely have fields added to it in a backward compatible way. E.g. the client requests `/healthz` with `Accept: application/json` and the server provides a structured response that can safely include the duration fields (and whatever we choose to add in the future). Client's wouldn't need to figure out how to parse the healthz text response to get machine readable access to the data. Would that be reasonable here?